apps/spruce/src/pages/task/taskTabs/TaskHistory/TaskTimeline/index.tsx

```diff
@@ -119,15 +135,83 @@ TaskTimeline.displayName = "TaskTimeline";
 
 export default TaskTimeline;
 
+const dateSeparatorHoverGroupStyles = css`
```

Comment:
Something in here can probably be adjusted because the dates are hidden when they're 2 boxes apart, but that's actually far enough to show both labels:

<img width="135" alt="Screenshot 2025-05-09 at 12 00 53 PM" src="https://github.com/user-attachments/assets/c5fb04e1-d486-4722-bf1a-e7b143426e09" />